src/graphql/schema.graphql

```diff
@@ -2654,6 +2655,10 @@ input TierInputType {
   password: String
   customFields: [CustomFieldsInputType]
 
+"""
+  Button text
+  """
+  button: String
```

Comment:
Looks like you're adding it twice, this is breaking.

Usually, this is automatically generated using `npm run graphql:get-schema`, not edited by hand.

Reply:
I added it to ` type Tier` and `input TierInputType` which I believe is for query and mutation. I had to do it manually because I couldn't setup the api locally on my windows laptop. This was the error message I got.

```
‼ request to http://localhost:3000/api/graphql failed, reason: connect ECONNREFUSED 127.0.0.1:3000
```

Reply:
Before running this command, frontend needs to be launched in another terminal using `npm run dev`.